_events/2019-10-17-toptal-try-catch.yml

```
@@ -0,0 +1,16 @@
+---
+title: Toptal Try/Catch: Experiências de Erros na Carreira
```

```suggestion
title: "Toptal Try/Catch: Experiências de Erros na Carreira"
```

Coloca entra aspas porque o jekyll ta capotando o corsa com o segundo `:` ele espera que seja outra chave.